src/listener/Color.php

```diff
@@ -22,10 +23,11 @@ class Color extends InlineListener
     /**
      * {@inheritDoc}
      */
-    public function process(Line $line)
+    public function process(Line $line, Lexer $lexer=null)
     {
         if (($color = $line->getAttribute('color'))) {
-            $this->updateInput($line, $this->ignore ? $line->input : '<span style="color:'.self::escape($color).'">'.$line->escapedInput().'</span>');
+            $color = ($lexer->escapeInput) ? self::escape($color) : $color;
```

Comment:
Escaping colors would be safe anyhow, no need to check whether escaping is enabled or not i would say.